VoodooSMBus/ELANTouchpadDriver.cpp

```diff
@@ -29,10 +29,69 @@
 
 OSDefineMetaClassAndStructors(ELANTouchpadDriver, VoodooSMBusSlaveDeviceDriver);
 
+void ELANTouchpadDriver::loadConfiguration() {
```

Comment:
I think this method is overly complicated. I would prefer to have a `loadBoolConfiguration(char* name)` and a `loadUInt64Configuration(char* name)` inside `helper` so we are able to reuse them inside other classes. Especially we should not need to iterate over the whole configuration. Do you want to make this change or should I?

Reply:
I don't have time so please, feel free to change whatever you want